app/assets/javascripts/components/campaign/campaign.jsx

```diff
@@ -30,13 +29,24 @@ export const Campaign = createReactClass({
       return <div />;
     }
     if (window.location.href.match(/overview/)) {
-      return <CampaignOverviewHandler {...this.props} />;
+      return (
+        <div>
+          <CampaignStats campaign={this.props.campaign} />
+          <div className="stats-download-modal">
+            <CampaignOverviewHandler {...this.props} />
+          </div>
+        </div>);
     }
+
+    let navBar;
+  if (window.location.href.match(/ores_plot/) || window.location.href.match(/alerts/)) {
+    navBar = (<CampaignNavbar
```

Comment:
Why not render the navbar for the overview tab?

Reply:
Navbar was rendered only for ores and Alerts, those two tabs are completely in React. For the rest of the tabs it comes from Rails views, this one is for an overview:
![navbar](https://user-images.githubusercontent.com/65791349/155582885-f76bfda8-dbb1-455a-8b05-41ff2c57a139.png).
I tried not to mess with stuff other than stats, but I could try to render React navbar for all tabs

Reply:
It doesn't need to be for all tabs right now, but I think it should be for the overview tab.